common_benchmark_suite/openxla/benchmark/models/jax/resnet50/resnet50_model.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from PIL import Image
+import io
+import jax.numpy as jnp
+import requests
+from transformers import AutoImageProcessor, FlaxResNetModel
+from typing import Any, Tuple
+
+from openxla.benchmark.models.jax import model_interfaces
+
+MODEL_NAME = "microsoft/resnet-50"
+
+
+def _get_image_input(width=224, height=224):
+  """Returns a sample image in the Imagenet2012 Validation Dataset.
+  Input size 224x224x3 is used, as stated in the MLPerf Inference Rules:
+  https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc#41-benchmarks
+  """
+  # We use an image of 5 applies since this is an easy example.
+  img_path = "https://storage.googleapis.com/iree-model-artifacts/ILSVRC2012_val_00000023.JPEG"
+  data = requests.get(img_path).content
+  img = Image.open(io.BytesIO(data))
+  img = img.resize((width, height))
+  return img
+
+
+class ResNet50(model_interfaces.InferenceModel):
```

Comment:
ResNets can come in different variants e.g. ResNet101. Similar to Bert and T5, let's rename this class to ResNet and include the `model_name` as a parameter, where it defaults to "microsoft/resnet-50".

Reply:
Done.